src/editor/dag.rs

```diff
@@ -1103,8 +1107,8 @@ mod tests {
     #[test]
     fn level_1_insertafter() {
         run_test_ok(
-            TestJSON::Array(vec![TestJSON::Array(vec![]), TestJSON::True]),
-            Path::from_vec(vec![0]),
+            TestJSON::Array(vec![TestJSON::True, TestJSON::True]),
+            Path::from_vec(vec![1]),
```

Comment:
I'm a bit confused by why these need to change - maybe I'm missing something?

Reply:
I just played with the test case, and forgot to change it back. But I think this one is clearer than the previous one.

Reply:
OK that makes sense - I was just wondering.